SpringProject/src/main/java/com/gradproject2019/users/repository/UserRepository.java

```diff
@@ -29,10 +30,12 @@ void updateUser(@Param("id") Long id,
                     @Param("email") String email,
                     @Param("occupation") String occupation);
 
-    @Query(value = "SELECT * FROM users where username = :username", nativeQuery = true)
+    @Query(value = "SELECT * FROM users WHERE username = :username", nativeQuery = true)
     Optional<User> findByUsername(@Param("username") String username);
 
-    @Query(value = "SELECT * FROM users where email = :email", nativeQuery = true)
+    @Query(value = "SELECT * FROM users WHERE email = :email", nativeQuery = true)
     Optional<User> findByEmail(@Param("email") String email);
 
+    @Query(value = "SELECT * FROM users WHERE first_name like CONCAT('%', :query,'%') OR last_name like CONCAT('%', :query, '%') ORDER BY last_name ASC", nativeQuery = true)
```

Comment:
nitpick: LIKE should be capitalized as is a MySql keyword
Can you format your queries like this
```
SELECT x,q,c 
FROM tablename
WHERE <>
AND <>
OR <>
ORDER BY foo
```
Its the standard way of formatting SQL

Reply:
also isn't what your doing more of a startswith? So the LIKE should be first_name 
`LIKE :query%`

Reply:
Have formatted it as suggested. For the 'startswith', i think the current way is more user-friendly since a user may misspell or not remember part of their manager's name, but still get the results they want